sprocs/variants_select.sql

```diff
@@ -20,9 +22,13 @@ BEGIN
         LEFT JOIN instance_questions AS iq ON (iq.id = v.instance_question_id)
         LEFT JOIN assessment_instances AS ai ON (ai.id = iq.assessment_instance_id)
         LEFT JOIN assessments AS a ON (a.id = ai.assessment_id)
-    WHERE v.id = variants_select.variant_id;
+    WHERE v.id = variants_select.variant_id
+          AND v.question_id = variants_select.question_id
```

Comment:
Per the docs for `variants_insert`, `question_id` can be null. I'm not sure if that's actually true anymore (I suspect it actually isn't), but that's probably worth spending a small amount of time looking into.

Reply:
Resolved in 9d7f050.